### ISSUE TITLE
Single User Event for multiple gRPC

### ISSUE THREAD

**Issue by drjdpowell** [ni/grpc-labview#201]
Currently, you have a separate User Event (UE) for each gRCP call type.   This requires a lot of extra "boilerplate" to set up each UE, as shown in this image:
![2022-12-06 13_02_20-route_guide_server lvlib_RouteGuide lvclass_Register gRPC Methods vi Block Diagr](https://user-images.githubusercontent.com/6072800/205920565-252ec89c-86cd-4b68-9404-98b239537444.png)

Normally, the value of separate UEs is strong typing, in that each can have a different type.  However, here all UEs have the same type, a single U64 that identifies the gRPC specific call.   Typedefs are used to convert, with no strong typing.  Thus one gains no benefit from all the extra boilerplate.

Instead, consider having only one User Event for all gRPC names, containing a cluster of two things: the U64 gRPC, plus a string containing the RPC name.   This single UE then carries all gRPC calls, with the string being fed into a case structure.

Then adding a new gRPC message just involves two message typedefs, a call to RegisterServerEvent (with tha existing UA), and a new frame in the case structure.

**Comment by drjdpowell** [comment]
Related to #199

**Comment by drjdpowell** [comment]
Actually, one can get close to what i would like, but combining all the UE into an array, and then looking up the Event Ref in a Map to get the RPC Name.   Like this:

![2022-12-07 12_11_52-JDP NewTest Server vi Block Diagram on JDP lvproj_My Computer _](https://user-images.githubusercontent.com/6072800/206176437-0e4324bc-ab07-4d98-a78e-1227255a8e36.png)

**Comment by drjdpowell** [comment]
See #206 for this example code

**Comment by JamesMc86** [comment]
I would disagree with this change. I think the ability to handle requests in different event loops may be necessary in many projects so different requests don't block each other (The async version of the current server template)
